research/conditional/train/configs/baselines/gpt/mot/generate_max_batch_size_report/base.yaml

```diff
@@ -0,0 +1,34 @@
+parent: research/conditional/train/configs/baselines/gpt/base.yaml
+md5_parent_hash: 1cc2f52ca625d4d24f327c4874413d0c
+interactive_debug: false
+time: 00:10:00
+n_gpus: 1
+
+
+params:
+    ^group_size: [2,4,8,16,32,64,128,256]
+    ^gradient_checkpointing: [true,false]
+    ^batch_size: [2,4,8,16,32,64,128,256]
+
+    flash_attention: true
+    n_steps: 3
+    learning_rate: 2e-3
+    n_experts: 512
+    ff_mode: cont_moe
+    flop_matched: true
+    sparsity_dim: 0
+    temperature: 1.0
+    name: memory_usage_grid_base
+    tags: [MoT, memory, base]
+    loss_checkpoint_chungs: 64
+    decoding_interval: 0
+    save_weights_interval: 0
+    logging_interval_heavy: 100000000000
+
+    use_dummy_dataset: true
+
+    init_type: "kaiming_uniform"
+    init_scale: 1.0
+
+    model_fits_filename: "results_base.json"
+    model_fits_params: "dmodel,dff,n_blocks,n_att_heads,group_size,gradient_checkpointing,batch_size"
```

Comment:
I do not understand this. Shouldn't the things we fit over be the ones we put in grid args? (^ delimiter above)

Reply:
I needed a way to distinguish different model sizes, therefore I have used "dmodel,dff,n_blocks,n_att_heads". Other args are the same as we grid over